.evergreen/.evg.yml

```diff
@@ -861,11 +861,35 @@ tasks:
           vars:
             file: ".evergreen/static-checks.sh"
 
-    - name: "test"
+    - name: "test-bson-and-crypt"
+      commands:
+        - func: "run tests"
+          vars:
+            TESTS: 'bson:test bson-kotlin:test bson-kotlinx:test bson-scala:test bson-record-codec:test mongodb-crypt:test'
+
+    - name: "test-core-and-legacy"
+      commands:
+        - func: "start-kms-kmip-server"
+        - func: "bootstrap mongo-orchestration"
+        - func: "run tests"
+          vars:
+            TESTS: 'driver-core:test driver-legacy:test'
```

Comment:
Collectively these take around 4 minutes to run on a replica set

Reply:
Seems a strange combo - would it make more sense for legacy tests to be in with sync tests?

Reply:
Logically, yes.  But mostly this is about balancing the test run times, and the sync tests are already the slowest at around 14 minutes.

Reply:
I decided to split them out to separate tasks